docs/scripting/script-server.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Script Core Features
+category: Tutorials
+tags:
+	- experimental
+---
+
+::: warning
+The Script API is currently in active development, and breaking changes are frequent. This page assumes the format of Minecraft beta 1.19.70
+:::
+
+In Scripting API, most of the core feaures are implemented in the `@minecraft/server` module, with lots of methods to interact a Minecraft world, including entities, blocks, dimensions, and more programmatically. This article contains a basic introduction to some of the core API mechanics, for more detailed infomation please visit [Microsoft docs](https://learn.microsoft.com/en-us/minecraft/creator/scriptapi/minecraft/server/minecraft-server).
+
+## Setup
+Script modules requires you will need to add the dependency into your `manifest.json`
```

Comment:
"Script modules requires you will need" looks like two different sentences combined, consider:
- Script modules require you to add the module as a dependency into your `manifest.json`
- You will need to add the Script module as a dependency in your `manifest.json`